doc/source/user_guide/plugins.txt

```diff
@@ -49,17 +49,25 @@ Any plugin in the ``_plugins`` directory is automatically examined by
 system::
 
   >>> import skimage.io as io
-  >>> io.plugins()
-  {'gtk': ['imshow'],
-   'matplotlib': ['imshow', 'imsave'],
-   'pil': ['imread'],
-   'qt': ['imshow'],
-   'test': ['imsave', 'imshow', 'imread']}
+  >>> io.find_available_plugins()
+  {'fits': ['imread', 'imread_collection'],
+   'freeimage': ['imread', 'imsave', 'imread_collection'],
+   'gdal': ['imread', 'imread_collection'],
+   'gtk': ['imshow'],
+   'imageio': ['imread', 'imsave', 'imread_collection'],
+   'imread': ['imread', 'imsave', 'imread_collection'],
+   'matplotlib': ['imshow', 'imread', 'imread_collection'],
+   'pil': ['imread', 'imsave', 'imread_collection'],
+   'qt': ['imshow', 'imsave', 'imread', 'imread_collection'],
+   'simpleitk': ['imread', 'imsave', 'imread_collection'],
+   'test': ['imsave', 'imshow', 'imread', 'imread_collection'],
+   'tifffile': ['imread', 'imsave', 'imread_collection']}
```

Comment:
I don't have `imageio` in my installation. Anyways, the full output of the command is kind of redundant for a documentation, so I'd suggest to keep those 5 most common plugins only. What do you think?